PFR/endgame.lean

```diff
@@ -63,14 +64,28 @@ local notation3 "I₁" => I[ U : V | S ]
 local notation3 "I₂" => I[ U : W | S ]
 
 /-- The quantity $I_3 = I[V:W|S]$ is equal to $I_2$. -/
-lemma I₃_eq : I[ V : W | S ] = I₂ := by sorry
+lemma I₃_eq : I[ V : W | S ] = I₂ := by
+  sorry
```

Comment:
I don't know the background material enough to know what the intended proof is here. I'm not sure how to turn X1' in the first expression into X1 in the second.

Reply:
OK.  You can relinquish it for now (by leaving a comment on the Zulip), and I will update the list to indicate the remaining task to establish here if you would like someone else to take over.

Reply:
The idea, by the way, is to use the symmetry $(X_1,X_1', X_2,X'_2) \mapsto (X'_1, X_1, X_2, X'_2)$, which preserves the distribution of the quadruple and also preserves $W$ and $S$, while interchanging $U$ and $V$.

Reply:
I made a small attempt at setting up these quadruples and writing `I` as a function on quadruples, but it's not there yet. I'll relinquish this for now.